scripts/ds-containerd-config-path-entry

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env sh
+
+CONTAINERD_CONFIG_DIR="/etc/containerd/config.toml"
+
+# check if config_path is defined for containerd registry plugin
+CONTAINERD_REGISTRY_CONFIG_PATH_DEFINED=$(sed -n '/^.*\[plugins\.\"io\.containerd\..*\"\.registry\]/,/^.*\[plugins\..*\]/p' "${CONTAINERD_CONFIG_DIR}" | grep -oE "^.*config_path += +[\"/A-Za-z\.]*" | grep -v "^\ *#")
+
+if [ "${CONTAINERD_REGISTRY_CONFIG_PATH_DEFINED}" ]; then
+	CONTAINERD_REGISTRY_CONFIG_PATH=$(echo ${CONTAINERD_REGISTRY_CONFIG_PATH_DEFINED} | tr -d '\"\|\ ' | awk -F "=" '{print $2}')
+	echo "Info: containerd registry plugin config_path: ${CONTAINERD_REGISTRY_CONFIG_PATH}"
+
+	# check REGISTRY_SERVICE_NODEPORT env variable is not empty
+	if [ ${REGISTRY_SERVICE_NODEPORT} ]; then
```

Comment:
Should this be quoted? I feel like your doing a bash trick I'm not familiar with.